lib/ruby_lsp_rails/rack_app.rb

```diff
@@ -3,32 +3,28 @@
 
 module RubyLsp
   module Rails
-    class Middleware
+    class RackApp
       extend T::Sig
 
       BASE_PATH = "/ruby_lsp_rails/"
 
-      sig { params(app: T.untyped).void }
-      def initialize(app)
-        @app = app
-      end
-
       sig { params(env: T::Hash[T.untyped, T.untyped]).returns(T::Array[T.untyped]) }
       def call(env)
         request = ActionDispatch::Request.new(env)
         path = request.path
-        return @app.call(env) unless path.start_with?(BASE_PATH)
 
         route, argument = path.delete_prefix(BASE_PATH).split("/")
 
         case route
+        when nil # e.g. GET /ruby_lsp_rails
```

Comment:
Just to provide a simple way to verify the server is running.

Reply:
We check if the server is running by invoking `activate`. When is this branch called?

Reply:
Only when we hit `http://localhost:3000/ruby_lsp_rails`, e.g. in the browser to allow for a manual check. I can drop if you prefer not to have.